htdocs/weapon_list.php

```diff
@@ -174,6 +174,12 @@ function applyFilter() {
 			case 3:
 				echo ('<td align="center" style="color: #06F;">Newbie</td>');
 			break;
+			case 4:
+				echo ('<td align="center" style="color: yellow;">Port</td>');
+			break;
+			case 5:
```

Comment:
Could you change these to using named constants please, it would make the code far more readable

Reply:
Done!